vaccine_feed_ingest/stages/load.py

```diff
@@ -181,13 +201,22 @@ def run_load_to_vial(
 
             continue
 
+    num_unknown_locations = (
+        num_imported_locations
+        - num_new_locations
+        - num_match_locations
+        - num_already_matched_locations
+    )
+
     logger.info(
-        "Imported %d source locations for %s (%d new, %d matched, %d unknown)",
+        "Imported %d source locations for %s "
+        "(%d new, %d matched, %d unknown, %d already matched)",
         num_imported_locations,
         site_dir.name,
         num_new_locations,
         num_match_locations,
-        num_imported_locations - num_new_locations - num_match_locations,
+        num_unknown_locations,
+        num_already_matched_locations,
```

Comment:
This is really "number of locations which we skipped because they were already matched and we configured the load step to skip matches, otherwise 0."

Could we change the log line slightly to reflect that?

Reply:
Updated log line to be conditional on if we are rematching or not